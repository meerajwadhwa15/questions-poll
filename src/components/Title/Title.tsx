import React, { FunctionComponent, ReactNode } from "react";

export type Props = {
  children: ReactNode;
};

const Title: FunctionComponent<Props> = ({ children }) => <h1 className="page-title">{children}</h1>;

export default Title;
