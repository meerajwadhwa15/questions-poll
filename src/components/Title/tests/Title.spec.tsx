import React from "react";
import Title from "../Title";
import { create } from "react-test-renderer";

const props = {
};

test("Matches the snapshot with Title", () => {
  const component = create(
      <Title {...props}>Page title</Title>
  );
  expect(component.toJSON()).toMatchSnapshot();
});
