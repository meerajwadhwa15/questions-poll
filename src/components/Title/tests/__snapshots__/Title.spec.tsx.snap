// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot with Title 1`] = `
<h1
  className="page-title"
>
  Page title
</h1>
`;
